File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Warehouses/IWarehouseData.cs
﻿using Ohtap.ECommerce.Domain.Models.Core.System;
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using Ohtap.ECommerce.Domain.Models.Objects.Stores;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Warehouses
{
    public interface IWarehouseData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState,IWarehouseEMail ,IWarehouseFax, IWarehousePhone
    {
        StoreBasic Store { get; set; }
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Warehouses/IWarehouseData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Warehouses;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IWarehouseData extends IWarehouseStore,IWarehouseAddress,IWarehouseEmail,IWarehouseFax,IWarehousePhone,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
